lib/std/crypto/aes.zig

```diff
@@ -5,7 +5,7 @@
 // and substantial portions of the software.
 // Based on Go stdlib implementation
 
-const std = @import("../std.zig");
+const std = @import("std");
```

Comment:
For now, the previous way to refer to the library is the preferred way. The only reason std has itself in its own package map is for this check: 

https://github.com/ziglang/zig/blob/a496f94be92581cdf97a1bd60c4dea0369e26395/lib/std/os.zig#L42

I'd like to keep it referring to std.zig because that is what user made packages will have to do and it's nice to keep the std lib code to match the same thing as an example.

If you like we can do a proposal for a special name to be used for a package to refer to itself. For example we already have `@import("root")` for referring to the application's root source file, but we could also introduce `@import("self")` which would refer to its own package. Then for the std lib we would no longer put "std" in the package table; we would rely on the "self" feature, which would be available to all packages. Are you interested in making that proposal?

Reply:
I like the idea. I'll write the proposal.